src/main/scripts/festerize/README.md

```diff
@@ -24,6 +28,12 @@ Usage: festerize [OPTIONS] SRC...
 
   Uploads CSV files to the Fester IIIF manifest service.
```

Comment:
I don't understand this paragraph and how it relates to the beginning of the file. It is possible I'm misreading the Github display. But I think this repeats the same heading as earlier in the readme.

Reply:
This is just a copy of what's at the beginning of the file (as well as the description in the Python script itself).

At some point we decided to put the entire "help" output inside the README, but whether or not that is useful is open for discussion.

Reply:
Yes, that's a question we should ask again... @emcaulay, we could just have the read me say, "See help output for use"... Is there value putting documentation on how to use the script in the README or would you prefer just a pointer to the "help" output?